### PR TITLE
blog: add release notes 3.11.0

### PR DESCRIPTION
**Pull request by kayx23**
Released on Oct 17.

### CODE REVIEW

**Comment by kayx23** [comment]
cc: @Yilialinn

File: blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+```shell
+curl -i "http://127.0.0.1:9180/apisix/admin/consumers" -X PUT \
+  -H "X-API-KEY: ${ADMIN_API_KEY}" \
+  -d '{
+    "username": "tom"
+  }'
+```
+
+To configure the consumer `key-auth` credential for `tom`, you can use leverage the credential object:
```
Comment (pottekkat) [inline blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md]:
```suggestion
To configure the consumer `key-auth` credential for `tom`, you can use the credential object:
```

File: blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+tags: [Community]
+---
+
+We are glad to present Apache APISIX 3.11.0 with exciting new features, bug fixes, and other improvements to user experiences.
+
+<!--truncate-->
+
+This new release adds a number of new features, including the addition of AI plugins to integrate with LLM providers, the support for AWS and GCP Secret Managers for secret management, and more.
+
+There are a few important changes included in this release. Should you find these changes impacting your operations, please plan accordingly for a seamless upgrade.
```
Comment (pottekkat) [inline blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md]: breaking changes instead of important changes?

**Review by pottekkat — DISMISSED** [review]

File: blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md (hunk 2 above)
Comment (kayx23) [reply blog/en/blog/2024/10/17/release-apache-apisix-3.11.0.md]:
```suggestion
There are a few breaking changes included in this release. Should you find these changes impacting your operations, please plan accordingly for a seamless upgrade.
```

**Review by yzeng25 — APPROVED** [review]

**Review by shreemaan-abhishek — APPROVED** [review]

**Pull request merged**